examples/dgmulti_2d/elixir_navier_stokes_convergence.jl

```diff
@@ -0,0 +1,206 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the ideal compressible Navier-Stokes equations
+
+get_Re() = 100
+get_Pr() = 0.72
+
+equations = CompressibleEulerEquations2D(1.4)
+# Note: If you change the Navier-Stokes parameters here, also change them in the initial condition
+# I really do not like this structure but it should work for now
+equations_parabolic = CompressibleNavierStokesEquations2D(equations, Reynolds=get_Re(), Prandtl=get_Pr(),
+                                                          Mach_freestream=0.5)
+
+# Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
+dg = DGMulti(polydeg = 3, element_type = Tri(), approximation_type = Polynomial(),
+             surface_integral = SurfaceIntegralWeakForm(flux_lax_friedrichs),
+             volume_integral = VolumeIntegralWeakForm())
+
+top_bottom(x, tol=50*eps()) = abs(abs(x[2]) - 1) < tol
+is_on_boundary = Dict(:top_bottom => top_bottom)
+mesh = DGMultiMesh(dg, cells_per_dimension=(16, 16); periodicity=(true, false), is_on_boundary)
+
+# Define initial condition
+# Note: If you change the parameters here, also change it in the corresponding source terms
+function initial_condition_navier_stokes_convergence_test(x, t, equations)
```

Comment:
Do we have a source for this NSE conv test? If it is @andrewwinters5000, I suggest to add something like
```suggestion
# This convergence test setup was originally derived by Andrew Winters (@andrewwinters5000)
function initial_condition_navier_stokes_convergence_test(x, t, equations)
```
here and to the other functions.

Reply:
Unfortunately, there is no reference for this manufactured solution.

Reply:
That is true. We can add the comment here that people can contact me if they are curious about this manufactured solution. Then we can update the comment later with a proper reference.